Lib/test/libregrtest/main.py

```diff
@@ -432,6 +433,26 @@ def display_header(self):
         print("== encodings: locale=%s, FS=%s"
               % (locale.getpreferredencoding(False),
                  sys.getfilesystemencoding()))
+        rl_version_cmd = """if 1:
```

Comment:
You may put the textwrap.dedent() directly here. But do you need "if 1:" in that case?